### ISSUE TITLE
Implement a way to set fail-fast on requests

### ISSUE THREAD

**Issue by stapelberg** [grpc/grpc-go#696]
https://github.com/grpc/grpc/blob/2a932daaafbec9874d9fb594419f68f9cb8f9187/doc/fail_fast.md specifies how fail-fast is supposed to work for all of gRPC.

In grpc-go, I can’t find a way to set fail-fast on requests. When searching through the code, I can find the boolean, but no line which sets it:

```
~/gocode/src/google.golang.org/grpc master $ ack failFast
call.go
145:		if lastErr != nil && c.failFast {

rpc_util.go
138:	failFast  bool
```

Am I missing something here? If not, can we track work on fail-fast in this issue please? :)

**Comment by iamqizhao** [comment]
The fail-fast semantics has not reached the full agreement across languages (still under discussion). I will roll it out once it is finalized.

**Comment by annismckenzie** [comment]
You can pass `grpc.FailFast(false)` to the client methods as the last argument (these are the call options). Because we're not ready to use the balancing feature we default to non-fail-fast for now.

**Comment by iamqizhao** [comment]
closed via #738
